java/client/src/main/java/glide/managers/ConnectionManager.java

```diff
@@ -202,6 +206,8 @@ private ConnectionRequest.Builder setupConnectionRequestBuilderGlideClusterClien
     private ConnectionRequestOuterClass.ReadFrom mapReadFromEnum(ReadFrom readFrom) {
         if (readFrom == ReadFrom.PREFER_REPLICA) {
             return ConnectionRequestOuterClass.ReadFrom.PreferReplica;
+        } else if (readFrom == ReadFrom.AZ_AFFINITY) {
```

Comment:
can we enforce https://github.com/valkey-io/valkey-glide/pull/2676/files#diff-fe2704ed4f33699f5f7faa1471f99351bb4edc0fc8119907a4f3d91e0d056c0aR472 through builders?